Created Cart management page

anushasunil opened the pull request:
![image](https://user-images.githubusercontent.com/90134948/154895668-325546ea-437f-4127-b0dc-b72fbbb13d7b.png)

anushasunil committed:
created utility class for positioning

anushasunil committed:
discarded redundant code

anushasunil committed:
created cart-page structure

anushasunil committed:
added styles to the components

anushasunil committed:
refactored media queries

anushasunil committed:
updated positioning of cards

anushasunil committed:
Merge branch 'product-listing' of https://github.com/anushasunil/bluebasket into cart-management

anushasunil committed:
Merge branch 'product-listing' of https://github.com/anushasunil/bluebasket into cart-management

anushasunil committed:
updated `nav` component

- added search button within input box
- updated link for logout button

anushasunil committed:
updated styles for the input box button

anushasunil committed:
Merge branch 'product-listing' of https://github.com/anushasunil/bluebasket into cart-management

coldpigli commented on Cart-management/cart-management.html:
```diff
@@ -0,0 +1,209 @@
+                <p>Copyright © 2022</p>
+                <p>All rights reserved</p>
+                <div class="social-links display-align-center">
+                   <a href="#" class="links"><i class="fa-brands fa-github"></i></a>
```
Suggestion to add social links.

anushasunil replied:
Thanks for pointing this out, will rectify it

dev-enforced commented on Cart-management/cart-management.css:
```diff
@@ -0,0 +1,264 @@
+    padding: 1rem 0;
+}
+
+li {
+    list-style: none;
+}
```
Can store this piece of code in utilities.css if it is used frequently

anushasunil replied:
yes, I'll do that

coldpigli reviewed (approved):
Code is good to merge. Everything is well structured.

dev-enforced reviewed (approved):
The code looks good to me!! The demonstration from image also looks clean!!
Good work @anushasunil

rittik-ghosh commented on Cart-management/cart-management.css:
```diff
@@ -0,0 +1,264 @@
+}
+
+button {
+    border-radius: 0;
```
The border radius is by default zero

anushasunil replied:
yes I agree but the component is taken from the component library and there I have initialized some border-radius so here it is nullifying the effect.

rittik-ghosh commented on Cart-management/cart-management.css:
```diff
@@ -0,0 +1,264 @@
+}
+
+.star-img {
+    color: orange;
```
can be put in a variable

anushasunil replied:
yes, I'll rectify it

rittik-ghosh reviewed (commented):
Have suggested a few changes. Overall the code is neat, clean and readable.

anushasunil committed:
refactored changes as per the review

- created utility class for `li` elements
- updated social link
- used variable
